app/services/sufia/workflow/workflow_by_admin_set_strategy.rb

```diff
@@ -0,0 +1,15 @@
+module Sufia
+  module Workflow
+    class WorkflowByAdminSetStrategy
+      def initialize(work, attributes)
+        @work = work
```

Comment:
Yes, you can remove it. and you can change the parameters to `def initialize(_work, attributes)` (this indicates that `_work` is not used)